### PR TITLE
Simplify ternary with elvis in site component

### PR DESCRIPTION
**Pull request by frankmayer**
### Summary of Changes
- Simplified some ternary operations with elvis operator

This PR is part of a set to try to separate some of the changes done in some of my previous batch PR's for site/components, which are still on hold (#12290, #12292, #12293, #12294).

### Testing Instructions

None, should not change behavior

### Documentation Changes Required

None.

### CODE REVIEW

**Comment by andrepereiradasilva** [comment]
I have tested this item :white_check_mark: successfully on 414f37825fd2752282485bf4c267ca8dc44bc6ba<br /><br />code review<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/13185">issues.joomla.org/tracker/joomla-cms/13185</a>.</sub>

**Comment by alikon** [comment]
I have tested this item :white_check_mark: successfully on 414f37825fd2752282485bf4c267ca8dc44bc6ba<br /><br />on code review<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/13185">issues.joomla.org/tracker/joomla-cms/13185</a>.</sub>

**Comment by jeckodevelopment** [comment]
RTC<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/13185">issues.joomla.org/tracker/joomla-cms/13185</a>.</sub>

**Comment by frankmayer** [comment]
Conflicts resolved. Could you pls review and merge if OK?

**Review by andrepereiradasilva — APPROVED** [review]

File: components/com_ajax/ajax.php [hunk 1]
```diff
@@ -82,7 +82,7 @@
 			$class = 'Mod' . ucfirst($module) . 'Helper';
 		}
 
-		$method = $input->get('method') ? $input->get('method') : 'get';
+		$method = $input->get('method') ?: 'get';
```
Comment (wilsonge) [inline components/com_ajax/ajax.php]: 👍

Comment (frankmayer) [inline components/com_ajax/ajax.php]: @matrikular Of course :smile:

**Pull request merged**